Tests/RunnerPart2/LimitsTest.php

```diff
@@ -279,93 +279,33 @@ public function dynamicBackendProvider(): Generator
     {
         yield 'no backend' => [
             'containerType' => null,
-            'expectedMemoryLimit' => '1000M',
+            'expectedMemoryLimit' => '500M',
+            'expectedCpuLimit' => '1',
+        ];
+        yield 'xsmall backend' => [
+            'containerType' => 'xsmall',
+            'expectedMemoryLimit' => '500M',
             'expectedCpuLimit' => '1',
         ];
         yield 'small backend' => [
-            'containerType' => null,
+            'containerType' => 'small',
```

Comment:
jak to, ze se tu najednou objevil ten container type? to se mi v tech zmenach nejak nedari vykoumat

Reply:
Protože ten test byl špatně - no backend měl containertype null a small taky (je to vstupní hodnota) - tj. byly to stejný casy i když jeden měl testovat nezadaný backend a jeden small